www/js/heatmap.js

```diff
@@ -183,4 +184,45 @@ angular.module('emission.main.heatmap',['ui-leaflet', 'emission.services'])
   $scope.selectCtrl = {}
   initSelect();
   $scope.getPopRoute();
-})
+
+
+  // Tour steps
+  var tour = {
+    config: {
+
+    },
+    steps: [{
+      target: '.datepicker',
+      content: 'This heatmap shows the aggregate data for all E-mission users. Select the dates you want to see, and filter by hours of the day (24h format) and days of the week. For example, if you enter 16 and 19 in the last field, and select Monday and Friday, you\'ll see the Heatmap filtered to show the traffic on weekdays between 4pm and 7pm.'
+    },
+    {
+      target: '.heatmap-mode-button',
```

Comment:
it has to be a class, I tried with id and it didn't work. That's why I added classes to some elements.